languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,21 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ELLES_VERBE_TOUTES" name="elles verbe toutes" default="temp_off">
```

Comment:
This rule causes too many FPs in the rule editor. Please always use the rule editor to make sure there are no false positives. Example found by the rule editor:

>Elles constituent tout un réseau de communications entre les mondes indien, iranien et celles de l'Asie centrale.